CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- GTM rules not importing `regexValidation` function.
```

Comment:
Maybe a test would be nice :)

Reply:
I think it was a fault on our linter more than in our test suite, since it was a simple syntax error instead of some error in a business logic. What do you think?